client/src/App.js

```diff
@@ -22,7 +22,7 @@ import {
 import { Tooltips } from "./tooltips";
 
 const add_definition_url = "https://docs.google.com/forms/d/e/1FAIpQLSfKF0yyleI5XdPVtl-bEuQUGy2HZPfnUU-e2sDjL31eLuygUA/viewform?usp=sf_link";
-const request_definition_url = "https://goo.gl/forms/xrZyTzaVo8Addq8d2";
+const request_definition_url = "/requestform";
```

Comment:
I want to hold off on this change - I think it would be nice to deploy both the request & definition forms to prod and test them out before switching the links over